swlor2_mdl/c_fcfield.mdl

```diff
@@ -0,0 +1,401 @@
+# NeverBlender 4.0.0|Blender 4.0.2|2023-12-30
```

Comment:
All models must be compiled.

Reply:
MDLs should be compiled, removed mdl that are in the creature pack